externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpBase.td

```diff
@@ -0,0 +1,43 @@
+//===-------------------------------------------------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TORCH_MLIR_DIALECT_TCP_BASE
+#define TORCH_MLIR_DIALECT_TCP_BASE
+
+include "mlir/IR/OpBase.td"
+
+def Tcp_Dialect : Dialect {
+  let name = "tcp";
+  let cppNamespace = "::mlir::torch::tcp";
+  let description = [{
+    Tensor Compute Primitives (TCP) dialect.
+
+    TCP is a mid-level transformation oriented IR for deep learning & similar
+    applications.
```

Comment:
Consider synchronizing this text with the definition of TCP in `externals/llvm-external-projects/torch-mlir-dialects/README.md`.

Reply:
Done